apps/homescreen/js/datastore.js

```diff
@@ -217,7 +218,6 @@
               cursor.continue();
             }
           };
-        txn.oncomplete = () => { resolve(results); };
```

Comment:
Purely for the unit test to pass - otherwise getAll completes before oncomplete is set, so oncomplete is never called (in practice, setting afterwards doesn't matter because it's async, but we make it sync in the unit tests). I guess I really ought to have changed the unit test really, but it'd really complicate the changeset vs. this one small change :)